docs/v1/P4Runtime-Spec.mdk

```diff
@@ -1173,7 +1173,9 @@ statically-assigned ID suffixes lead to non-unique IDs (&ie; if the P4
 programmer tries to assign the same ID suffix to two different P4 objects of the
 same type by annotating them with the same `@id` value). Note that it is not
 possible for the P4 programmer to change the value of the 8-bit ID prefix, which
-encodes the object type.
+encodes the object type. The programmer is free to include or not include the
+8-bit prefix; the compiler will add it if not already present, and will check
```

Comment:
I know what you mean by "include or not include the 8-bit prefix", but perhaps nicer to be slightly more precise in a spec, e.g. "The programmer is free to leave the 8-bit prefix as 0, in which case the compiler will replace the 0 with the correct value for the kind of object the annotation is annotating.  The programmer may also fill in the 8-bit prefix with a non-0 value, in which case the compiler will give an error if the 8-bit prefix does not contain the correct value, or leave it as is if it is correct."

Reply:
Done, thanks for the great suggestion.